.storybook/preview.js

```diff
@@ -0,0 +1,19 @@
+import { TwitchContext } from '../src/components/TwitchContext';
+import React from 'react';
+
+export const parameters = {
+  actions: { argTypesRegex: '^on[A-Z].*' },
+};
```

Comment:
What does this do?

Reply:
Actually part of the default, I haven't looked into it yet

Reply:
https://storybook.js.org/docs/react/essentials/actions Seems to be related to making a log for actions like onClick when used in the interface. Looks sensible to leave this alone if we make something reliant on actions.